### DISCUSSION TITLE
Fitur Jadwal, Pengisi kajian, Imam Tarawih & Nama Penyedia Takjil, dan Pendataan zakat fitrah

### DISCUSSION THREAD

**Discussion by aahedi** [buku-masjid/buku-masjid#7] [Ideas]
Assalamu'alaikum,

Hai, saya pengurus DKM di salah satu masjid.
Saya mengapresiasi apa yg Saudara kerjakan dalam membantu masjid.

Saya teringat salah satu kesulitan di DKM adalah menentukan tanggal puasa hari dan jadwal imam tarawih & jadwal penyedia takjilnya.

Termasuk jadwal pengisi kajian dan nama² penyedia takjil pengajiannya. Biasanya saya buat satu tahun penuh dari syawal sampai ramadhan. Apakah sudah ada fitur tersebut?

Terakhir pendataan untuk pemberi zakat fitrah berikut data kategori: pakai uang berapa atau beras berapa KG nya? dan total penerimaan keseluruhan.
Apakah sudah ada fitur tersebut?

Jika belum ada, semoga bisa menjadi bahan masukan.

Terima kasih
wassalamu'alaikum

**Comment by nafiesl** [comment]
Waalaikumussalam mas, @aahedi.

Alhamdulillah, terima kasih masukannya mas. Untuk fitur tersebut di sistem ini belum ada, baru fitur jadwal pengajian rutin dan khatib. Sebelumnya fitur yang mas sebutkan di atas memang belum terpikirkan oleh kami. Dan ini menjadi masukan penting.

Baik kita definisikan secara umum, fitur yang kira-kira diperlukan:

1. Penanggalan bulan hijriyah secara manual untuk bulan puasa
2. Jadwal imam tarawih (dan mungkin pengisi kultum)
3. Daftar penyedia takjil pengajian
4. Fitur penerimaan dan distribusi zakat fitrah.

Usulan ini kami catat dulu untuk pengembangan selanjutnya, mungkin kami perlu diskusi lebih detail dengan mas dan atau beberapa DKM lain untuk merumusukan bagaimana alur kerja fitur-fiturnya.

Karena fitur-fitur ini dibutuhkan bulan ramadhan, insyaallah kita masih punya banyak waktu untuk pengembangannya.

Untuk saat ini kami masih fokus di dokumentasi dulu, untuk mempermudah user menggunakan fitur yang sudah ada.

Jazakallahu khairan, mas @aahedi.

**Comment by nafiesl** [comment]
Assalamualaikum mas @aahedi, bulan depan kita rencana mau mulai rancang dan membuat fitur kegiatan ramadhan, zakat fitrah dan Qurban. Apakah ada kontak telegram/whatsapp yang bisa dihubungi mas? Ijin, saya ingin kontak mas untuk diskusi contoh dokumen dan laporannya sebagai referensi.

Atau jika mas berkenan bisa sapa saya di telegram: https://t.me/nafiesl.

Terima kasih mas.
